planning/behavior_path_avoidance_module/include/behavior_path_avoidance_module/scene.hpp

```diff
@@ -64,7 +65,13 @@ class AvoidanceModule : public SceneModuleInterface
   std::shared_ptr<AvoidanceDebugMsgArray> get_debug_msg_array() const;
 
 private:
-  bool isSatisfiedSuccessCondition(const AvoidancePlanningData & data) const;
+  /**
+   * @brief return the result whether the module can stop path generation process.
+   * @param avoidance data.
+   * @return [first]  it will be true when the ego avoid all obstacles.
+   *         [second] it will be true when all obstacles have gone.
+   */
+  std::pair<bool, bool> isSatisfiedSuccessCondition(const AvoidancePlanningData & data) const;
```

Comment:
IMO, using `std::pair` is difficult to understand about output type.
It seems that `first` and `second` will be `true` exclusively.
So, to clarify the meaning of output, I suggest to define `enum class` like following:

```
enum class AvoidanceStatus {
  RUNNING = 0,
  SUCCESS,
  CANCEL
};
```

How do you think?
(It's maybe also okay to add these members to `AvoidanceState`)

Reply:
@rej55 Thanks for your comment. It seems to be good idea, and I fixed in https://github.com/autowarefoundation/autoware.universe/pull/7142/commits/7b4ea26b6d21a40320b756002cfad03811a3e8bc.